src/app/technical-record/vehicle-summary/vehicle-summary.component.spec.ts

```diff
@@ -39,15 +39,13 @@ describe('VehicleSummaryComponent', () => {
   });
 
   it('should check if axles has no parking brake mrk', () => {
-    expect(component.formatVehicleClassDescription()).toEqual('The first letter should be capital');
+    expect(component.formatVehicleClassDescription()).toEqual(
+      'The first letter should be capital'
+    );
   });
 
   it('should create format the vehicle class description', () => {
     fixture.detectChanges();
     expect(component.vehicleClassDescription).toEqual('The first letter should be capital');
   });
```

Comment:
for this test please use this `  it('should create format the vehicle class description', () => {
    fixture.detectChanges();
    expect(component.vehicleClassDescription).toEqual('Motorbikes over 200cc');
  });`